tests/system/actions.go

```diff
@@ -13,13 +14,24 @@ type akResult struct {
 	returnCode int
 }
 
+func splitToArgs(cmdArgs string) []string {
+	re := regexp.MustCompile(`\w+=".*?"|\w+=[\w.:]+|"[^"]+"|[\w-.:/]+`)
+	args := re.FindAllString(cmdArgs, -1)
+	for i, arg := range args {
+		if len(arg) >= 1 && arg[0] == '"' && arg[len(arg)-1] == '"' {
+			args[i] = strings.Trim(arg, "\"")
+		}
+	}
+	return args
+}
+
 func runAction(t *testing.T, akPath, akAddr, step string) (any, error) {
 	match := actions.FindStringSubmatch(step)
 	switch match[1] {
 	case "setenv":
 		return nil, setEnv(strings.TrimSpace(strings.TrimPrefix(step, match[1])))
 	case "ak":
-		args := append(config.ServiceUrlArg(akAddr), strings.Fields(match[3])...)
+		args := append(config.ServiceUrlArg(akAddr), splitToArgs(match[3])...)
```

Comment:
instead of sophisticated hard to maintain regex you could try strings.FieldsFunc

look at this example
https://stackoverflow.com/questions/47489745/splitting-a-string-at-space-except-inside-quotation-marks

Reply:
I prefer to use regexp actually, for me it's simpler. but let's hear what @itayd and @daabr think

Reply:
I love regexes, but I have to admit there are reasons why they're less ideal here:

1. There are standard, popular, and well-tested alternatives (see Efi's link)
2. The explanation of this short regex requires 4 lines of comments - that's a code smell for regex complexity
3. The less we assume/require, the better; in other words, future CLI changes and even small-but-reasonable tweaks in the expected behavior will make it much harder to read/maintain (e.g. supporting more than `\w`, supporting spaces around `=`, specifying URLs with all their complexity in the CLI, etc.)

Reply:
That said, it seems to me that https://go.dev/play/p/Ed4IV97L7H is better than `strings.FieldsFunc` - we don't want to preserve `""`

Reply:
I'll go for csv reader. By default it doesn't allow quotes in parsed fields, but it seems there is LazyQuotes flag, which allows to parse things like `c="meow"` which we are using in our tests